src/config/index.ts

```diff
@@ -3,10 +3,10 @@ import setup from './parts/setup'
 import disables from './parts/disables'
 import nuxt from './parts/nuxt'
 
-export function createBasicNuxtConfig(options: NuxtESLintConfigOptions = {}): FlatConfig[] {
+export function createBasicNuxtConfig(options: NuxtESLintConfigOptions): FlatConfig[] {
```

Comment:
The reason those are optional is because they are designed to also be part of the public API, meaning that if users don't want to use the generated config, they can still use it as a normal factory like:

```ts
import { createBasicNuxtConfig } from 'nuxt-module-eslint-config/config'

export default [
   ...createBasicNuxtConfig()
]
```

Reply:
Certainly makes sense, will push an update to fix this.